client/cmd/dexc/main.go

```diff
@@ -127,3 +137,10 @@ func main() {
 	wg.Wait()
 	ui.Close()
 }
+
+func disconnectActive() {
+	for _, activeConnectionMaster := range activeConnectionMasters {
+		activeConnectionMaster.Disconnect()
+	}
+	os.Exit(1)
```

Comment:
As a general rule of thumb, never call `os.Exit` except in a stripped down main that wraps another function returning an error.  When you call `os.Exit` anywhere, the app is dead right then.  The `return` after `disconnectActive` is not called, and neither are any other statements like the crucial `wg.Wait` and `ui.Close`.

Reply:
I've removed the use of os.Exit for the rpc and web server error cases. Other calls to os.Exit still remain for config errors and core startup errors. I was not positive of the implications of changing those at the moment.

Reply:
Those are fine for now.  This app needs a little refactoring to allow all return paths to return an error code to the OS.